storage/storage-interface/src/lib.rs

```diff
@@ -570,6 +570,38 @@ impl MoveStorage for &dyn DbReader {
 /// expected of an Aptos DB. This adds write APIs to DbReader.
 #[allow(unused_variables)]
 pub trait DbWriter: Send + Sync {
+    /// Get a (stateful) state snapshot receiver.
+    ///
+    /// Chunk of accounts need to be added via `add_chunk()` before finishing up with `finish_box()`
+    fn get_state_snapshot_receiver(
```

Comment:
@JoshLind - You might want to rebase as this API has changed, we no longer expose `AccountStateBlob` in the DbWriter anymore.